lib/src/disposable/transformer/transformations.dart

```diff
@@ -169,10 +169,17 @@ void transformLibraryToImportDartAsync(
       directive.uri.stringValue == 'dart:async');
   if (hasDartAsyncImport) return;
 
+  // If this file is a part (meaning the imports are defined in a separate,
+  // parent file), then nothing can be done.
+  final isPartOf =
+      unit.directives.any((directive) => directive is PartOfDirective);
```

Comment:
Can we warn or something if we run into a part? Seems like we should be a little noisy about it without actually failing to prevent surprises.